crops/crop.go

```diff
@@ -115,3 +124,36 @@ func (c Crop) ComputeConsequences(event interface{}) consequences.Results {
 	r := consequences.Results{IsTable: false, Result: ret}
 	return r
 }
+func (c Crop) computeImpactedCase(e hazards.ArrivalandDurationEvent) float64 {
+	// Determine damage percent based on damage dur curve and event dur
+	perdmg := c.lossFunction.ComputeDamagePercent(e)
+	fmt.Println("damage percent is ", perdmg)
+	croploss := (perdmg / 100) * c.GetValuePerOutputUnit()
+	fmt.Println("Crop loss is : ", croploss)
+	// value added to field before loss by production
+	fmt.Println("total pruduction costs are:", c.productionFunction.productionCostLessHarvest)
+	fmt.Println("fixed costs are: ", c.productionFunction.GetfixedCosts())
+	loss := croploss + c.productionFunction.productionCostLessHarvest
+	fmt.Println("total loss is:", loss)
+	if loss > c.valuePerOutputUnit {
+		// Throw some error
+	}
+	return 10
+}
+func (c Crop) computeDelayedCase() float64 {
+	// delayed loss is equivalent to total marketable value less harvest cost, times the percent loss due to late planting
+	// Not using interpolated % loss for late plant
+	valuelessharvest := (c.GetValuePerOutputUnit()) - c.productionFunction.harvestCost
+	croploss := (valuelessharvest) * c.productionFunction.lossFromLatePlanting
+	fmt.Println("total loss is : ", croploss)
+	return croploss
+}
+
+func (c Crop) computeNotPlantedCase() float64 {
+	// Assume Loss is only fixed costs
+	return c.productionFunction.GetfixedCosts()
+}
+
+func (c Crop) computeSubstitueCase(e hazards.ArrivalandDurationEvent) float64 {
```

Comment:
this is orphaned  - need to hook it up to the select case

Reply:
hooked compute method up to proper case. Found another case was using wrong method as well.  commit 95ff9bd70eb474b50fae3facb1794251c2a34544